ui/src/wizard/ResourceGroupCreateStep.ts

```diff
@@ -20,7 +20,7 @@ export class ResourceGroupCreateStep<T extends types.IResourceGroupWizardContext
         // tslint:disable-next-line:no-non-null-assertion
         const newLocation: string = wizardContext.location!.name!;
         const resourceClient: ResourceManagementClient = createAzureClient(wizardContext, ResourceManagementClient);
-        const rgExists: boolean = await resourceClient.resourceGroups.checkExistence(newName);
+        const rgExists: boolean = (await resourceClient.resourceGroups.checkExistence(newName)).body;
```

Comment:
What's with the `.body` here?

Reply:
The previous API call returned a Promise<boolean>, the new one returns a Promise<ResourceGroupsCheckExistenceResponse>, which has a body of boolean.

Reply:
That feels like a bug in the sdk